juicy-html.html

```diff
@@ -36,14 +38,26 @@
             }
         };
         /**
+         * Skips/disregards pending request if any
+         * @return {XMLHttpRequest} skipped XHR request, in fact just `this.pending`
+         */
+        JuicyHTMLPrototype.skip = function() {
```

Comment:
Yes "keeping it public" sorry.

I don' like `abort...` as we are not aborting the loading of external file, what about `skipStampingPendingFile`?

Personally, I don't like YAGNI approach, especially in the world where we don't have any requirements and all we do is forseeing potential applications.
Also from functional, mathematical point of view I prefer functions that renurns something. They are harder to test and comprehens.

However if you are really on it, I may remeve this `return`